requirements.txt

```diff
@@ -0,0 +1,2 @@
+ansible
+jenkins-job-builder
```

Comment:
Could you dump the contents of constraints.txt here and just use this file for `-c` and `-r` to pip?  I vaguely recall something like that being done in openstack-ansible.